content/articles/2018-05-dplyr-0-7-5.Rmd

```diff
@@ -0,0 +1,161 @@
+---
+title: dplyr 0.7.5
+slug: dplyr-0.7.5
+description: >
+    dplyr 0.7.5 has reached CRAN. This is mostly a bugfix release with two important new changes.
+date: 2018-05-25
+author: Kirill Müller, Romain François, Hadley Wickham
+photo:
+  url: https://unsplash.com/photos/aqrIcYonB-o
+  author: Phúc Long
+categories: [package]
+---
+
+
+```{r setup, include = FALSE}
+options(crayon.enabled = TRUE)
+options(pillar.bold = TRUE)
+
+library(dplyr)
+knitr::opts_chunk$set(collapse = TRUE, comment = pillar::style_subtle("#>"))
+
+colourise_chunk <- function(type) {
+  function(x, options) {
+    sprintf('<pre class="knitr %s">%s</pre>\n',
+      tolower(options$engine),
+      fansi::sgr_to_html(htmltools::htmlEscape(x))
+    )
+  }
+}
+
+knitr::knit_hooks$set(
+  output = colourise_chunk("output"),
+  message = colourise_chunk("message"),
+  warning = colourise_chunk("warning"),
+  error = colourise_chunk("error")
+)
+```
+
+We’re excited to announce version 0.7.5 of the *dplyr* package, the grammar of data manipulation in the tidyverse.  This minor release features the move to *tidyselect*, minor features like scoped operations on grouped data frames and support for raw vectors, and a number of bug fixes.  Please see the [release notes](https://github.com/tidyverse/dplyr/releases/tag/v0.7.5) for the full list of improvements and bug fixes.
```

Comment:
Having "minor...features" twice in that second sentence is a bit confusing; possibly change the first "features" to "includes", or cut the second "minor features" entirely: "the move to tidyselect, scoped operations on grouped data frames, support for raw vectors, and a number of bug fixes"